exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80){
+  return "A"
+} if (mark < 80 && mark > 60)
```

Comment:
This works but lets try to simplify it - if by the time this line is reached, all marks >= 80 have been dealt with (return was called), then do we need to check again if mark < 80?)